sklearn/preprocessing/tests/test_label.py

```diff
@@ -513,3 +513,14 @@ def test_inverse_binarize_multiclass():
                                                    [0, 0, 0]]),
                                        np.arange(3))
     assert_array_equal(got, np.array([1, 1, 0]))
+
+
+def test_empty_labels():
```

Comment:
please use a meaningful name, e.g., test_label_encoder_empty_array and move it above (where tests for LabelEncoder exists).